libafl_libfuzzer/Cargo.toml

```diff
@@ -25,7 +25,6 @@ rustversion = "1.0"
 
 [features]
 #! ## Feature Flags
-document-features = ["dep:document-features"]
```

Comment:
Shouldn't be here, right?

Reply:
Ah it doesn't have the cfg-attr, probably I forgot it - see:
https://github.com/AFLplusplus/LibAFL/blob/7d2c854b714cd636d4875369ea8509967ea9abe4/libafl/src/lib.rs#L6

Reply:
Anyway please add that line as well, main fails because I forgot :) 
```rust
error[E0433]: failed to resolve: use of undeclared crate or module `document_features`
  --> libafl_libfuzzer/src/lib.rs:73:10
   |
73 | #![doc = document_features::document_features!()]
   |          ^^^^^^^^^^^^^^^^^ use of undeclared crate or module `document_features`
   ```

Reply:
I just made it not optional -- no reason to, anyways.

Reply:
Fixed in #1480